src/views/GraphDetail.vue

```diff
@@ -378,6 +419,7 @@ export default Vue.extend({
 
       this.nodeTypes = graph.nodeTables;
       this.edgeTypes = [graph.edgeTable];
+      // eslint-disable-next-line no-underscore-dangle
       this.nodes = nodes.nodes.map((d) => d._id);
```

Comment:
FWIW, eslint allows you to [provide exceptions](https://eslint.org/docs/rules/no-underscore-dangle#allow) to this rule, so we could add `_id` to that if we wanted.

Reply:
Yeah but this is literally the one time it comes up so I thought a one-off silencing was simpler.